iff -ur drumstick-0.3.1/tests/vpiano/CMakeLists.txt drumstick-0.3.1-fix-implicit-linking/tests/vpiano/CMakeLists.txt
--- drumstick-0.3.1/utils/vpiano/CMakeLists.txt	2010-04-19 16:35:34.000000000 +0200
+++ drumstick-0.3.1-fix-implicit-linking/utils/vpiano/CMakeLists.txt	2010-05-28 23:44:10.000000000 +0200
@@ -59,6 +59,10 @@
     drumstick-alsa
 )
 
+IF(UNIX)
+TARGET_LINK_LIBRARIES(drumstick-vpiano X11)
+ENDIF(UNIX)
+
 INSTALL(TARGETS drumstick-vpiano
         RUNTIME DESTINATION bin)